lib/src/invariant.dart

```diff
@@ -34,16 +32,16 @@ Position getCoord(dynamic coord) {
 }
 
 ///
-/// Unwrap coordinates from a Feature, Geometry Object or an Array
+/// Unwrap coordinates from a Feature, Geometry Object or a List
 ///
 /// @name getCoords
-/// @param {Array<any>|Geometry|Feature} coords Feature, Geometry Object or an Array
-/// @returns {Array<any>} coordinates
+/// @param {List<dynamic>|Geometry|Feature} coords Feature, Geometry Object or a List
+/// @returns {List<any>} coordinates
 /// @example
-/// var poly = turf.polygon([[[119.32, -8.7], [119.55, -8.69], [119.51, -8.54], [119.32, -8.7]]]);
+/// var poly = Polygon(coordinates: [[Position(119.32, -8.7), Position(119.55, -8.69), Position(119.51, -8.54), Position(119.32, -8.7)]]);
```

Comment:
the same applies here: refactor to Dart doc
and put the comment in a block like this: ` ```dart `, like in line 12